tools/smithy-rs-tool-common/src/changelog.rs

```diff
@@ -7,15 +7,64 @@
 
 use anyhow::{bail, Context, Result};
 use serde::{de, Deserialize, Deserializer, Serialize};
-use std::fmt;
+use std::fmt::{self, Display};
 use std::path::Path;
 use std::str::FromStr;
 
+#[derive(Copy, Clone, Debug, Serialize, PartialEq, Eq)]
+pub enum SdkAffected {
+    Client,
+    Server,
+    Both
```

Comment:
Thinking about this more, we should probably call this `All` in case there's more than client and server in the future.